.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
 /vendor/
 tools
+testing/kuttl/generated/
```

Comment:
⛏️ This matches correctly, but do you mind adding a slash `/` at the front of the pattern? It reminds me that this pattern is anchored to the directory of the gitignore file. 🙇🏻 

⛏️ Please also include the trailing newline in this and other files.

Reply:
Added / and newline (found a few extra files that needed them)